control/setup_control.go

```diff
@@ -468,3 +474,9 @@ func UpdateTrapzBlock(ctx context.Context, name string, maxVel float64, dependsO
 	}
 	return nil
 }
+
```

Comment:
maybe we can move the tuning check to live here? That way the same code can be reused for the motor case.

```suggestion

// CheckTuning compares two sets of PIDConfigs to determine if tuning was performed
func CheckTuning(controlledName string, configPIDVals, tunedVals []PIDConfig) error {
	// if loop has been tuned but the values haven't been added to the config, error with tuned values
	if configPIDVals[0].NeedsAutoTuning() || configPIDVals[1].NeedsAutoTuning() {
		templateString := "%v has been tuned, please copy the following control values into your config: [%v]"
		signalStrings := ""
		numTunedSigs := 0
		for sigNum, pidConfig := range configPIDVals {
			if pidConfig.NeedsAutoTuning() {
				if (tunedVals)[sigNum].NeedsAutoTuning() {
					return fmt.Errorf("PID tuning for component %v is in progress", controlledName)
				}
				if numTunedSigs > 0 {
					signalStrings += ","
				}
				signalStrings += TunedPIDErr((tunedVals)[sigNum])
				numTunedSigs++
			}
		}
		return fmt.Errorf(templateString, controlledName, signalStrings)
	}
	return nil
}

```

Reply:
error logs for both tuning:
<img width="892" alt="Screenshot 2024-09-06 at 5 29 09 PM" src="https://github.com/user-attachments/assets/4c5e6184-34b0-44b9-b00b-f0d2c52f3994">

error log for single tuning: 
<img width="995" alt="Screenshot 2024-09-06 at 5 30 27 PM" src="https://github.com/user-attachments/assets/bc70de92-778e-4c99-ad37-7bc730be9bb9">

error log for tuning in progress(note i changed the wording after screencap):
<img width="380" alt="Screenshot 2024-09-06 at 5 30 42 PM" src="https://github.com/user-attachments/assets/83aa2a77-667b-4c96-bd06-5663003ee7d3">